app/javascript/packages/document-capture/components/address-search.tsx

```diff
@@ -14,31 +16,49 @@ interface Location {
 
 interface AddressSearchProps {
   onAddressFound?: (location: Location) => void;
-  registerField: (field: string) => Ref<HTMLInputElement>;
+  registerField?: (field: string) => Ref<HTMLInputElement> | undefined;
```

Comment:
Dunno if you deleted a comment, but I arrived here from an email notification about this type assignment. I think it might be better to defer the type to how it's defined by `@18f/identity-form-steps`:

```ts
import type { RegisterFieldCallback } from '@18f/identity-form-steps';

interface AddressSearchProps {
  // ...
  registerField?: RegisterFieldCallback;
}
```

Reply:
(It was your comment at https://github.com/18F/identity-idp/pull/7502#discussion_r1051144577, maybe the GitHub UI was being slow to update)